src/routes/chart-page/index.jsx

```diff
@@ -0,0 +1,12 @@
+import Chart from '../../components/chart/chart'
```

Comment:
Could you configure the base path in `jsconfig.json`? If you can then raise a separate PR for it, also make sure to create an issue for it.

```suggestion
import Chart from 'components/chart/chart'
```

Reply:
Never done that before, so no ..... 
And there is multiple folder (component/routes/asset) we need to access though....

Reply:
I'll take care of it. Thanks 👉